client/bottom-slot.js

```diff
@@ -69,6 +69,7 @@ module.exports = function ({ config={}, guruResult, customSetup }={}) {
 
 	// show banner
 	if (customSetup) {
+		if (customSetup.default) customSetup = customSetup.default; // ESM modules
```

Comment:
I believe the inline syntax without brackets make it more exposed to involuntary breakages due to change to multilines so I would opt for the more traditional and known syntax...
```suggestion
		if (customSetup.default) { 
		     customSetup = customSetup.default; // ESM modules
		 }
```
...but is a personal opinion, surely won't block on this 😄

Reply:
i'm actually of the same opinion, so changing it to bracketed :)